src/pages/overview/pdf-electronic-seal-api/howtos/electronic-seal-api.md

```diff
@@ -829,101 +842,112 @@ public class ElectronicSealWithAppearanceOptions {
                     .WithProviderName(providerName)
                     .WithCredentialID(credentialID)
                     .WithPin(pin)
                     .WithCSCAuthContext(cscAuthContext)
                     .Build();
-                
-                
-                //Create SealingOptions instance with all the sealing parameters.
-                SealOptions sealOptions = new SealOptions.Builder(certificateCredentials, fieldOptions)
-                    .WithAppearanceOptions(appearanceOptions).Build();
-
-                //Create the PDFElectronicSealOperation instance using the SealOptions instance
-                PDFElectronicSealOperation pdfElectronicSealOperation = PDFElectronicSealOperation.CreateNew(sealOptions);
-
-                //Set the input source file for PDFElectronicSealOperation instance
-                pdfElectronicSealOperation.SetInput(sourceFile);
 
-                //Set the optional input seal image for PDFElectronicSealOperation instance
-                pdfElectronicSealOperation.SetSealImage(sealImageFile);
-
-                //Execute the operation
-                FileRef result = pdfElectronicSealOperation.Execute(executionContext);
-
-                // Save the output at specified location.
-                result.SaveAs(output/sealedOutput.pdf);
+                // Create parameters for the job
+                PDFElectronicSealParams pdfElectronicSealParams =
+                    PDFElectronicSealParams.PDFElectronicSealParamsBuilder(certificateCredentials, fieldOptions)
+                        .WithDocumentLevelPermission(documentLevelPermission)
+                        .WithAppearanceOptions(appearanceOptions)
+                        .Build();
+
+                // Creates a new job instance
+                PDFElectronicSealJob pdfElectronicSealJob = new PDFElectronicSealJob(asset, pdfElectronicSealParams);
+                pdfElectronicSealJob.SetSealImageAsset(sealImageAsset);
+
+                // Submits the job and gets the job result
+                String location = pdfServices.Submit(pdfElectronicSealJob);
+                PDFServicesResponse<PDFElectronicSealResult> pdfServicesResponse =
+                    pdfServices.GetJobResult<PDFElectronicSealResult>(location, typeof(PDFElectronicSealResult));
+
+                // Get content from the resulting asset(s)
+                IAsset resultAsset = pdfServicesResponse.Result.Asset;
+                StreamAsset streamAsset = pdfServices.GetContent(resultAsset);
+
+                // Creating output streams and copying stream asset's content to it
+                String outputFilePath = "/output/sealedOutput.pdf";
+                new FileInfo(Directory.GetCurrentDirectory() + outputFilePath).Directory.Create();
+                Stream outputStream = File.OpenWrite(Directory.GetCurrentDirectory() + outputFilePath);
+                streamAsset.Stream.CopyTo(outputStream);
+                outputStream.Close();
             }
```

Comment:
please add .NET sample for Apply Electronic Seal with trusted timestamp on PDF

Reply:
Fixed